Key Update Limit

hannestschofenig opened the pull request:
See errata 8050: https://www.rfc-editor.org/errata/eid8050

hannestschofenig committed:
Key Update Limit

See errata 8050: https://www.rfc-editor.org/errata/eid8050

ekr commented on draft-ietf-tls-rfc9147bis.md:
```diff
@@ -2173,15 +2173,21 @@ With a 128-bit key as in AES-128, rekeying 2^64 times has a high
+sending and receiving implementations MAY instead enforce an epoch
+limit of 2^16-1.  In this case, the implementation MUST check for
+this limit, if reached, terminate the association. In some cases, it
+is otherwise possible for the epoch number to reach 2^16+1.
```
@hannestschofenig I think you're just saying that if you *don't* make this change, then the epoch number can exceed 2^16. Is it fine here to just change this to 2^16 where Frosne flags?

ekr replied:
@hannestschofenig?

hannestschofenig replied:
Changed to  2^16.

hannestschofenig committed:
Update draft-ietf-tls-rfc9147bis.md

ekr committed:
Clean up PR#302

I reread David's errata and realized my original comments
were wrong. I've cleaned up the text a bit to hopefully
make it clearer and avoid other people misreading the text
the way I did.
